README.md

```diff
@@ -49,6 +49,21 @@ $.ajax({
     });
 ```
 
+## Allow default to all ajax requests
+
+In this example, there is default retry to all ajax requests, for 5 seconds between retries
+
+      $.ajaxSettings.
+            shouldRetry= function() {
```

Comment:
Please put this on the above line, and we're missing a space after `=`

Reply:
Fixed